digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -157,35 +64,44 @@ const environmentalSensor = new EnvironmentalSensor('environmentalSensor', prope
 const deviceInformation = new DeviceInformation('deviceInformation');
 const modelDefinition = new ModelDefinition('urn_azureiot_ModelDiscovery_ModelDefinition', null, modelDefinitionHandler);
 
-const deviceClient = DeviceClient.fromConnectionString(process.env.DEVICE_CONNECTION_STRING, Mqtt);
-
 const capabilityModel = 'urn:azureiot:samplemodel:1';
 
 async function main() {
-  const digitalTwinClient = new DigitalTwinClient(capabilityModel, deviceClient);
-  digitalTwinClient.addInterfaceInstance(environmentalSensor);
-  digitalTwinClient.addInterfaceInstance(deviceInformation);
-  digitalTwinClient.addInterfaceInstance(modelDefinition);
-  await digitalTwinClient.register();
+  // mqtt is implied in this static method
+  digitalTwinClient = DigitalTwinClient.fromConnectionString(process.env.DEVICE_CONNECTION_STRING);
```

Comment:
>fromConnectionString [](start = 40, length = 20)

Should the first parameter be the capability model?

Reply:
yes.

---
In reply to: [401191616](https://github.com/Azure/azure-iot-sdk-node/pull/738#discussion_r401191616) [](ancestors = 401191616)